src/main.rs

```diff
@@ -48,7 +48,8 @@ impl std::fmt::Display for RoundState {
     write!(
       f, 
       "The round was {:}.",
-      if self == &RoundState::Lose { "lost" } else if self == &RoundState::Draw { "a draw" } else { "won" } )
+      if self == &RoundState::Lose { "lost" } else if self == &RoundState::Draw { "a draw" } else { "won" } 
```

Comment:
nepouzivas rust-analyzer? respektive rustfmt, nie som si isty ci je to automaticky includenute. Taketo commity nemaju existovat. O to sa ma starat tooling automaticky. Ak si naaahodou nieco zle spravil, amendni to v predchadzajucom commite.